packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart

```diff
@@ -288,25 +297,25 @@ class OwnUserResponse {
         devices: Device.listFromJson(json[r'devices']),
         id: mapValueOfType<String>(json, r'id')!,
         image: mapValueOfType<String>(json, r'image'),
-        invisible: mapValueOfType<bool>(json, r'invisible') ?? false,
+        invisible: mapValueOfType<bool>(json, r'invisible')!,
         language: mapValueOfType<String>(json, r'language')!,
         lastActive: mapDateTime(json, r'last_active', r''),
         latestHiddenChannels: json[r'latest_hidden_channels'] is Iterable
             ? (json[r'latest_hidden_channels'] as Iterable).cast<String>().toList(growable: false)
             : const [],
-        mutes: UserMute.listFromJson(json[r'mutes']),
+        mutes: UserMuteResponse.listFromJson(json[r'mutes']),
         name: mapValueOfType<String>(json, r'name'),
-        online: mapValueOfType<bool>(json, r'online') ?? false,
-        privacySettings: PrivacySettings.fromJson(json[r'privacy_settings']),
-        pushNotifications: PushNotificationSettings.fromJson(json[r'push_notifications']),
+        online: mapValueOfType<bool>(json, r'online')!,
+        privacySettings: PrivacySettingsResponse.fromJson(json[r'privacy_settings']),
+        pushNotifications: PushNotificationSettingsResponse.fromJson(json[r'push_notifications']),
         revokeTokensIssuedBefore: mapDateTime(json, r'revoke_tokens_issued_before', r''),
         role: mapValueOfType<String>(json, r'role')!,
         teams: json[r'teams'] is Iterable
             ? (json[r'teams'] as Iterable).cast<String>().toList(growable: false)
             : const [],
-        totalUnreadCount: mapValueOfType<int>(json, r'total_unread_count') ?? 0,
-        unreadChannels: mapValueOfType<int>(json, r'unread_channels') ?? 0,
-        unreadThreads: mapValueOfType<int>(json, r'unread_threads') ?? 0,
+        totalUnreadCount: mapValueOfType<int>(json, r'total_unread_count')!,
+        unreadChannels: mapValueOfType<int>(json, r'unread_channels')!,
+        unreadThreads: mapValueOfType<int>(json, r'unread_threads')!,
```

Comment:
Please make sure all these values exist before scrapping the defaults

Reply:
Tested those on dogfooding, works fine